mod/05_standard/set/type/uri.rb

```diff
@@ -0,0 +1,11 @@
+format do
+  include Phrase::Format
+
+  view :core do |args|
+    build_link _render_raw(args), render_title(args) || card.name
+  end
+
+  view :uri do |args|
+    build_link _render_raw(args)
```

Comment:
It doesn't have any meaning for non-uri types.  If you think otherwise, tell me what it would do?  What would the link be targeted at?  This is what I'm saying.  The core view for most types is not a link, but it is for this type.  So, the url_link view is a link for this type because the target is the raw content of my card. exactly like the core view, it's just that I also want that content in the <a> element content.  There isn't really an equivalent to this for other types.  Core may not be right, maybe raw, but I don't see what other option would give something that makes sense.